src/elements/kw-projects-header/kw-projects-header.html

```diff
@@ -757,6 +916,13 @@
             _openModal () {
                 this.fire('show-modal');
             },
+            _assignArtProgress (user) {
+                let artProgress = user && user.profile && user.profile.stats['kano-draw'] &&
+                        user.profile.stats['kano-draw'].groups && user.profile.stats['kano-draw'].groups.basic &&
+                        user.profile.stats['kano-draw'].groups.basic.challengeNo || null;
+                //Minus one for zero-based numbering
+                this['art_basic'] = artProgress - 1 || 0;
```

Comment:
I'm not sure whether I'm missing something here but the `artProgress` variable will be either true or null and then you subtract - 1 from it?